js/templates/modals/connectionManagement/configurationForm.html

```diff
@@ -0,0 +1,94 @@
+<div class="contentBox padMd clrP clrBr clrSh3">
+  <div class="flexVCent" style="height: 35px">
```

Comment:
Since either way isn't great, I would just tend to leave in what's in place. If you really feel strongly about it, please feel free to submit a PR :)